fe/fe-core/src/main/java/org/apache/doris/external/elasticsearch/EsUtil.java

```diff
@@ -403,16 +404,15 @@ public static Type toDorisType(String esType) {
             case "double":
             case "scaled_float":
                 return Type.DOUBLE;
+            case "date":
```

Comment:
I think you should use ScalarType.getDefaultDateType(Type.DATE) to enable users to read date as datev2 type